### PR TITLE
ui: VAULT-8673 Add empty states to certificates and roles

### PR DESCRIPTION
**Pull request by kiannaquach**
**Description**
- Add empty states for roles and certificates
- Add test for empty states

**Roles**
Screen Shot 2023-01-13 at 12 03 38 PM
**Certificates**
Screen Shot 2023-01-13 at 12 03 43 PM

### CODE REVIEW

File: ui/lib/pki/addon/templates/roles/index.hbs [hunk 1]
```diff
@@ -16,50 +16,58 @@
...
-    <LinkedBlock class="list-item-row" @params={{array "roles.role.details" pkiRole.id}} @linkPrefix={{this.mountPoint}}>
-      <div class="level is-mobile">
-        <div class="level-left">
-          <div>
-            <Icon @name="user" class="has-text-grey-light" />
-            <span class="has-text-weight-semibold is-underline">
-              {{pkiRole.id}}
-            </span>
+{{#if this.model.hasConfig}}
+  {{#if this.model.roleModel.length}}
```
Comment (hashishaw) [inline ui/lib/pki/addon/templates/roles/index.hbs:20]: nit: the name `roleModel` makes it sound like we expect this to be a singular model, when really it's the list of roles. Same with `certificateModel` naming. Personally I'd prefer something more indicative of what the item is, like `roles` or `roleList`.

**Review by hashishaw — APPROVED** [review]
Great work! One small non-blocking comment about naming. Thank you!
